arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -28,40 +40,54 @@
 
 	qcom,board-id = <42 0>;
 
+	vreg_touch_vdd: regulator-touch-vdd {
+		compatible = "regulator-fixed";
+		regulator-name = "touch_vdd";
+		startup-delay-us = <4000>;
+		enable-active-high;
+		regulator-boot-on;
+		gpio = <&tlmm 72 0>;
+	};
+
 	reserved-memory {
-		adsp_mem: adsp@95100000 {
-			reg = <0x0 0x95100000 0x0 0x2600000>;
+		adsp_mem: adsp@95700000 {
+			reg = <0x0 0x95700000 0x0 0x2600000>;
 			no-map;
 		};
 
-		wlan_msa_mem: wlan_msa@97700000 {
-			reg = <0x0 0x97700000 0x0 0x180000>;
+		wlan_msa_mem: wlan_msa@97d00000 {
+			reg = <0x0 0x97d00000 0x0 0x180000>;
 			no-map;
 		};
 
-		npu_mem: npu@97880000 {
-			reg = <0x0 0x97880000 0x0 0x80000>;
+		npu_mem: npu@97e80000 {
+			reg = <0x0 0x97e80000 0x0 0x80000>;
 			no-map;
 		};
 
-		ipa_fw_mem: ipa_fw@97900000 {
-			reg = <0x0 0x97900000 0x0 0x10000>;
+		ipa_fw_mem: ipa_fw@97f00000 {
+			reg = <0x0 0x97f00000 0x0 0x10000>;
 			no-map;
 		};
 
-		ipa_gsi_mem: ipa_gsi@97910000 {
-			reg = <0x0 0x97910000 0x0 0x5000>;
+		ipa_gsi_mem: ipa_gsi@97f10000 {
+			reg = <0x0 0x97f10000 0x0 0x5000>;
 			no-map;
 		};
 
-		gpu_mem: gpu@97915000 {
-			reg = <0x0 0x97915000 0x0 0x2000>;
+		gpu_mem: gpu@97f15000 {
+			reg = <0x0 0x97f15000 0x0 0x2000>;
 			no-map;
 		};
 
-		rmtfs_mem: rmtfs@fde01000 {
+		tz_mem: trust-zone@86200000 {
```

Comment:
Start address is same as sm7150 dts.
```
		tz_mem: trust-zone@86200000 {
			reg = <0x0 0x86200000 0x0 0x2d00000>;
			no-map;
		};
```

Use & to refer to a node (at the beginning of the file, look at davinci's dts):
&tz_mem {
	reg = <0x0 0x86200000 0x0 0x4900000>;
};